pkg/protocol/multipart_test.go

```diff
@@ -69,11 +71,167 @@ Content-Type: application/json
 		t.Fatalf("unexpected error: %s", err)
 	}
 
-	if err := WriteMultipartForm(&w, form, "foo"); err != nil {
+	// boundary is too long
+	err = WriteMultipartForm(&w, form, s)
```

Comment:
The length of boundary is in the range of [1,70], which can be verified for strings outside this range.

Reply:
done, thx~